.github/workflows/on-pullrequest-powershell.yml

```diff
@@ -24,6 +24,8 @@ jobs:
     uses: Ed-Fi-Alliance-OSS/Ed-Fi-Actions/.github/workflows/powershell-analyzer.yml@main
     permissions:
       security-events: write
+      actions: read
+      contents: read
```

Comment:
This workflow is [currently failing](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Actions/actions/runs/13090939516) because these permissions are not granted.